src/init.cpp

```diff
@@ -988,6 +988,9 @@ bool AppInitParameterInteraction()
     if (GetArg("-rpcserialversion", DEFAULT_RPC_SERIALIZE_VERSION) < 0)
         return InitError("rpcserialversion must be non-negative.");
 
+    if (GetArg("-rpcserialversion", DEFAULT_RPC_SERIALIZE_VERSION) > 1)
```

Comment:
Which would imply the default is not always the max. It is not clear from the comments if this is desired.